src/main/java/org/aksw/iguana/cc/storage/impl/TriplestoreStorage.java

```diff
@@ -79,10 +81,14 @@ public void storeResult(Model data) {
 		//submit Block to Triple Store
 		UpdateProcessor processor = UpdateExecutionFactory
 				.createRemote(blockRequest, endpoint, createHttpClient());
-		try {
+		if (MainController.Args.dryRun) {
```

Comment:
Please add a comment that explains this branching. Otherwise, it will be hard to recall what it was good for in the future.